wee_slack.py

```diff
@@ -2973,6 +3046,8 @@ def linkify_text(message, team):
             named = targets.groups()
             if named[1] in ["group", "channel", "here"]:
                 message[item[0]] = "<!{}>{}".format(named[1], named[2])
+            elif named[1] in list(usergroups.viewkeys()):
+                message[item[0]] = "<!subteam^{}|{}>".format(usergroups[named[1]], named[1])
```

Comment:
Did you see my comment about including `named[2]` here?

> If you write e.g. colon after the group, it will be lost because named[2] is not included. I see this was an issue with group/channel/here too, I've fixed that in master now.

Reply:
I did but I must overlooked something when I was testing.